numerics/fast_sin_cos_cycle.hpp

```diff
@@ -0,0 +1,11 @@
+﻿
+#pragma once
+
+namespace principia {
+namespace numerics {
+
+//TODO(phl):comment
+void FastSinCosCycle(double x, double& sin, double& cos);
```

Comment:
I'm not sure I like the word "fast" here, after all we always try to write a fast implementation for the precision that we target; we wouldn't call a correctly-rounded Pochhammer symbol "SlowPoch".
Here we are aiming for a precision that is essentially @stephentyrone's definition of a ["fistful of ULPs"](https://twitter.com/stephentyrone/status/975813418736340993)...